src/data/faq.json

```diff
@@ -316,7 +316,15 @@
                         "anchor": "minimum_requirements",
                         "active": true,
                         "textblock": [
-                            "<b>UPDATE</b><br/><a href='https://www.coronawarn.app/en/blog/2020-12-15-ios-12-5' target='_blank' rel='noopener noreferrer'>Telekom and SAP to make Corona Warn app available for iOS 12.5.</a><hr/>",
+
+                            "<b>UPDATE 8th February 2021</b><br/>As of February 5, 2021, only the Corona-Warn-App (CWA) versions 1.5 and later are supported. For this reason, users with older versions will receive a notification to update their app accordingly. This applies to both iOS and Android. At the same time, however, certain minimum requirements apply to the operating system: Android at least 6.0 and iOS at least 13.7.",
+                            "iOS users who have previously used iOS 13.5, for example, must now update the iOS operating system to at least iOS 13.7 in order to continue using the CWA. Only then the technically correct operation of the app can be ensured.",

```

Comment:
Only then the technically correct operation of the app can be ensured.
=>
Only then can the technically correct operation of the app be ensured. (Word order)

Reply:
ok